### PR TITLE
Chore: go1.20

### PR DESCRIPTION
**Pull request by shuker85**
Checklist:

* [x] My organization is added to [USERS.md](https://github.com/argoproj/argo-events/blob/master/USERS.md).

Fixes: https://github.com/argoproj/argo-events/issues/2484

### CODE REVIEW

File: USERS.md [hunk 1]
```diff
@@ -47,3 +47,4 @@ Organizations below are **officially** using Argo Events. Please send a PR with
 1. [Yubo](https://www.yubo.live/)
 1. [WooliesX](https://wooliesx.com.au/)
 1. [Woolworths Group](https://www.woolworthsgroup.com.au/)
+1. [MariaDB](https://mariadb.com/)
```
Comment (whynowy) [inline USERS.md:50]: Can you do it in alphabetical order?

**Review by whynowy — COMMENTED** [review]
Thanks!

File: USERS.md (hunk 1 above)
Comment (shuker85) [reply USERS.md]: Done

**Review by whynowy — APPROVED** [review]
Thanks!

**Pull request merged**